### PR TITLE
db/climate-demo-kpi

### PR DESCRIPTION
**Pull request by sebastianbochan**
- [x] - redesign cells / rows
- [x] - upgrade bindings / PoC KPI
- [x] - add KPI for default series
- [x] - refresh KPI when change city
- [x] - KPI data
- [x] - interactions (click events)
   - [x] - navigator
   - [x] - map
   - [x] - chart
 - [x] - states
    - [x] - hover state
    - [x] - current state
 - [x] - navigator
 - [x] - TX as default

### CODE REVIEW

File: samples/dashboard/demos/climate/demo.js [hunk 1]
```diff
@@ -199,30 +204,17 @@ async function setupDashboard() {
                             dataPool
                                 .getStore(city)
                                 .then(store => {
-                                    const data = store.table.modified.getRows(
-                                        void 0, void 0,
-                                        ['time', dataScope]
-                                    );
+                                    dataScope = 'TX';
```
Comment (karolkolodziej) [inline samples/dashboard/demos/climate/demo.js:207]: If I'm looking at something else (for example rain) and I click on another city I don't get the data I needed (rain).
To me, it is a bit confusing. What do you think?

File: ts/Dashboard/Actions/Bindings.ts [hunk 2]
```diff
@@ -101,16 +115,49 @@ class Bindings {
             fireEvent(component, 'mount');
         }
 
-            if (cell && component) {
-                component.setCell(cell);
-                cell.mountedComponent = component;
+        if (cell && component) {
```
Comment (karolkolodziej) [inline ts/Dashboard/Actions/Bindings.ts:118]: Shouldn't we also add a `selected` state? IMO after clicking on some KPI, the selection color should be changed.

File: ts/Dashboard/Actions/Bindings.ts [hunk 3]
```diff
@@ -133,6 +180,9 @@ class Bindings {
             case 'DataGrid': 
                 component = DataGridComponent.fromJSON(json as DataGridComponent.ClassJSON);
                 break;
+            // case 'kpi':
```
Comment (karolkolodziej) [inline ts/Dashboard/Actions/Bindings.ts:183]: Remove?

**Review by karolkolodziej — CHANGES_REQUESTED** [review]

File: ts/Dashboard/Actions/Bindings.ts (hunk 3 above)
Comment (sebastianbochan) [reply ts/Dashboard/Actions/Bindings.ts:183]: Its intentional to do not miss it. 

`toJSON` method should be added in `KPI component`. I will create a separate ticket in Asana.

File: ts/Dashboard/Actions/Bindings.ts (hunk 2 above)
Comment (sebastianbochan) [reply ts/Dashboard/Actions/Bindings.ts:118]: Its active state is added in the click event (a few lines) above.

**Review by bre1470 — APPROVED** [review]
KPI's look good. Something is not right with the map.

File: samples/dashboard/demos/climate/demo.js (hunk 1 above)
Comment (sebastianbochan) [reply samples/dashboard/demos/climate/demo.js:207]: We adjust it when the final design will come ;)

**Review by pawellysy — APPROVED** [review]
Looks good! Thanks!

**Review by karolkolodziej — APPROVED** [review]

**Pull request merged**